app/code/community/Lesti/Fpc/Model/Observer.php

```diff
@@ -212,10 +212,13 @@ protected function _prepareLayout(
         array $dynamicBlocks
     )
     {
-        $xml = simplexml_load_string(
-            $layout->getXmlString(),
-            Lesti_Fpc_Helper_Data::LAYOUT_ELEMENT_CLASS
-        );
+        $xml = $layout->getNode();
+        if (!is_a($xml, Lesti_Fpc_Helper_Data::LAYOUT_ELEMENT_CLASS)) {
```

Comment:
Yeah, I wasn't sure at first, but after a local test I can confirm that the if is really superfluous. Gonna push a new commit in a few secs.